frontend/src/utils/getOverflowMenuButtons/getOverflowMenuButtons.tsx

```diff
@@ -0,0 +1,15 @@
+import { ReactElement } from 'react';
+
+/**
+ * Returns all of the buttons to be displayed in the toolbar overflow menu.
+ * @param {Array<ReactElement | false>} buttons - All of the buttons that could be rendered
+ * @param {number} toolbarButtonsDisplayed - The number of buttons displayed on the toolbar, any excess are displayed in the overflow menu
+ * @returns {Array<ReactElement | null | false>} - The buttons for the toolbar overflow menu
```

Comment:
Same nit here.  Do we really need both `null` and `false`?

Reply:
We don't need `null`. We do need `false`, though.